src/procstat.c

```diff
@@ -872,11 +890,27 @@ static ssize_t reset_u64_series(void *object, uint64_t arg, char *buffer, size_t
 	return 1;
 }
 
+static ssize_t reset_interval_u64_series(void *object, uint64_t arg, char *buffer, size_t length)
+{
+	struct procstat_series *series_stat = object;
+	struct procstat_series_u64 *series = series_stat->private;
+	uint32_t control;
```

Comment:
maybe rename `control` to `reset_interval`?

Reply:
If you insist. I used the same pattern as in reset_u64_series()

Reply:
i'm not insisting, just proposing :)
at the end it is your call ;)